pwndbg/commands/checksec.py

```diff
@@ -26,4 +26,4 @@ def checksec(file=None):
         if program:
```

Comment:
This command should probably just call `pwndbg.checksec.checksec(file)`.

For the error message - it might be good to raise an exception in `checksec(..)` and just catch it here.